src/containers/subjects/IGV/genomes.ts

```diff
@@ -1,57 +1,68 @@
+// More info see https://github.com/igvteam/igv.js/wiki/Reference-Genome
+
 export const genomes = [
   {
     id: 'hg38',
     name: 'Human (GRCh38/hg38)',
-    fastaURL: 'https://s3.amazonaws.com/igv.broadinstitute.org/genomes/seq/hg38/hg38.fa',
-    indexURL: 'https://s3.amazonaws.com/igv.broadinstitute.org/genomes/seq/hg38/hg38.fa.fai',
-    cytobandURL: 'https://s3.amazonaws.com/igv.org.genomes/hg38/annotations/cytoBandIdeo.txt.gz',
+    fastaURL: 'https://igv-genepattern-org.s3.amazonaws.com/genomes/seq/hg38/hg38.fa',
+    indexURL: 'https://igv-genepattern-org.s3.amazonaws.com/genomes/seq/hg38/hg38.fa.fai',
+    cytobandURL: 'https://igv-genepattern-org.s3.amazonaws.com/genomes/hg38/cytoBandIdeo.txt.gz',
+    aliasURL: 'https://igv-genepattern-org.s3.amazonaws.com/genomes/hg38/hg38_alias.tab',
+    chromSizesURL: 'https://hgdownload.soe.ucsc.edu/goldenPath/hg38/bigZips/hg38.chrom.sizes',
+    twoBitURL: 'https://hgdownload.soe.ucsc.edu/goldenPath/hg38/bigZips/hg38.2bit',
     tracks: [
       {
         name: 'Refseq Genes',
         format: 'refgene',
-        url: 'https://s3.amazonaws.com/igv.org.genomes/hg38/ncbiRefGene.txt.gz',
+        url: 'https://hgdownload.soe.ucsc.edu/goldenPath/hg38/database/ncbiRefSeq.txt.gz',
         indexed: false,
         visibilityWindow: -1,
         removable: false,
         order: 1000000,
+        infoURL: 'https://www.ncbi.nlm.nih.gov/gene/?term=$$',
```

Comment:
Oh, these igv.js reference genomes have changed. Nice update there, Will.!

It seems to load a tad quicker than previous ones. Nice. (FYI @jrobinso). Was thinking mirroring local region ap-southeast with #87